### PR TITLE
Add some instructions on running with GPU rendering on headless, rootless servers

### PR DESCRIPTION
**Pull request by jeasinema**

### CODE REVIEW

File: docs/source/notes/performance-tips.rst [hunk 1]
```diff
@@ -27,4 +27,39 @@ All credits go to Tencent researchers who kindly shared this piece of informatio
...
+Docker images for headless rendering with GPU
+------------------------------------------------
+
+The above instructions might not work with a server without root access. You may use `this <https://github.com/ehfd/docker-nvidia-egl-desktop>`_ prebuilt docker (an alternative can be found `here <https://github.com/MineDojo/egl-docker>`_) instead. These docker images are brought to you by `Seungmin Kim <https://github.com/ehfd>`_, `Xiaojian Ma <https://github.com/jeasinema>`_ and the `MineDoJo <https://minedojo.org>`_ team.
+
+To begin with, pull & run this docker on your server. Please make sure your container is running using ``docker ps``
+
+.. code-block:: bash
+	
+	docker run --gpus 1 -it -e TZ=UTC -e SIZEW=1920 -e SIZEH=1080 -e REFRESH=60 -e DPI=96 -e CDEPTH=24 -e PASSWD=mypasswd -e WEBRTC_ENCODER=nvh264enc -e BASIC_AUTH_PASSWORD=mypasswd -p 8080:8080 ghcr.io/ehfd/nvidia-egl-desktop:latest
```
Comment (Miffyli) [inline docs/source/notes/performance-tips.rst]: Just small clarification: what are the passwords "mypasswd" used for if for anything? Are they required for using the image you have created, or are they used to login to the docker in some way (ps: rookie docker user here 🙃 ).

Comment (jeasinema) [reply docs/source/notes/performance-tips.rst]: that's a good one. Actually, I'm working on a minimal `Dockerfile` that could get rid of all this. Will update this PR real soon.

**Comment by jeasinema** [comment]
@Miffyli I've updated the doc with the new minimal docker file.

**Review by Miffyli — APPROVED** [review]
LGTM! Thanks a ton! Now the instructions are super-duper clear. :)

**Pull request merged**